fastly/auth.py

```diff
@@ -1,9 +1,12 @@
 """
 """
+import sys
+if sys.version_info[0] == 2:
```

Comment:
`< 3`  I know it won't actually work with python 1, but it's better form to specify the version using an inequality.